frame/assets/src/lib.rs

```diff
@@ -241,7 +243,8 @@ pub mod pallet {
 			+ Copy
 			+ MaybeSerializeDeserialize
 			+ MaxEncodedLen
-			+ TypeInfo;
+			+ TypeInfo
+			+ CheckedNeg;
```

Comment:
Avoid adding here - just add it into `BaseArithmetic` trait/blanket impl along with all the other `Checked*` bounds.